src/Menu/NotificationManager.php

```diff
@@ -184,6 +185,25 @@ public function display_aggregated_notification_pill(): void {
 		}
 	}
 
+	/**
+	 * Filter the notification count for the admin menu.
```

Comment:
```suggestion
	 * Filters the notification count for the admin menu.
```

Use third-person singular verb.